static/app/components/replays/player/scrubber.tsx

```diff
@@ -7,34 +7,70 @@ import * as Progress from 'sentry/components/replays/progress';
 import {useReplayContext} from 'sentry/components/replays/replayContext';
 import {divide, formatTime} from 'sentry/components/replays/utils';
 import {space} from 'sentry/styles/space';
+import toPercent from 'sentry/utils/number/toPercent';
+import useOrganization from 'sentry/utils/useOrganization';
 
 type Props = {
   className?: string;
 };
 
 function Scrubber({className}: Props) {
-  const {currentHoverTime, currentTime, replay, setCurrentTime} = useReplayContext();
-  const durationMs = replay?.getDurationMs();
+  const {currentHoverTime, currentTime, replay, setCurrentTime, timelineScale} =
+    useReplayContext();
+
+  const organization = useOrganization();
+  const hasNewTimeline = organization.features.includes('session-replay-new-timeline');
+
+  const durationMs = replay?.getDurationMs() ?? 0;
 
   const percentComplete = divide(currentTime, durationMs);
   const hoverPlace = divide(currentHoverTime || 0, durationMs);
 
+  const initialTranslate = 0.5 / timelineScale;
+
+  const starting = percentComplete < initialTranslate;
+  const ending = percentComplete + initialTranslate > 1;
+
+  const translate = () => {
+    if (starting) {
+      return 0;
+    }
+    if (ending) {
+      return 1 - 2 * initialTranslate;
+    }
+    return currentTime > durationMs ? 1 : percentComplete - initialTranslate;
+  };
+
   return (
     <Wrapper className={className}>
+      {className?.includes('PlayerScrubber') && hasNewTimeline ? (
+        <ZoomIndicatorContainer style={{left: toPercent(translate()), top: '-10px'}}>
+          <ZoomTriangleDown />
+          <ZoomIndicator />
+        </ZoomIndicatorContainer>
+      ) : null}
+      {className?.includes('PlayerScrubber') && hasNewTimeline ? (
+        <ZoomIndicatorContainer
+          style={{left: toPercent(translate() + 2 * initialTranslate), top: '-2px'}}
+        >
+          <ZoomIndicator />
+          <ZoomTriangleUp />
+        </ZoomIndicatorContainer>
+      ) : null}
```

Comment:
```suggestion
      {className?.includes('PlayerScrubber') && hasNewTimeline ? (
        <ZoomIndicatorContainer style={{left: toPercent(translate()), top: '-10px'}}>
          <ZoomTriangleDown />
          <ZoomIndicator />
        </ZoomIndicatorContainer>
        <ZoomIndicatorContainer
          style={{left: toPercent(translate() + 2 * initialTranslate), top: '-2px'}}
        >
          <ZoomIndicator />
          <ZoomTriangleUp />
        </ZoomIndicatorContainer>
      ) : null}
```
you can combine the results since the conditions are the same

Reply:
It needs to have 1 parent element, and if i put it in a div, it doesn't work :(

Reply:
`<Fragment>`?